learn/pkg/learn/questionfm.go

```diff
@@ -72,3 +84,21 @@ func (f *questionFrontmatter) Unseal(privateKey string) error {
 	f.Answer = unsealed
 	return nil
 }
+
+func validateGeneratedQuestions(gen string) error {
+	if gen == "" {
+		return nil
+	}
+	questions := splitTrim(gen)
+	if len(questions) == 1 && questions[0] != "all" {
+		return fmt.Errorf("%w: more than one question generated for generate-questions", ErrInvalidFrontmatter)
```

Comment:
nit: add "must" - "more that one question must be generated..."

Reply:
done.